packages/@okta/i18n/src/properties/login.properties

```diff
@@ -1006,14 +1006,33 @@ oie.try.again = Try again
 
 ## Device Enrollment Terminal
 enroll.title.oda = Download Okta Verify
+enroll.title.oda.with.account = Additional setup required to use Okta FastPass
+enroll.title.oda.without.account = Set up an Okta Verify Account
```

Comment:
Can you un-capitalize "Account"? So it should say "Set up an Okta Verify account" since we typically use sentence casing. This would also match the mocks